src/screens/LandingPage/index.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react'
+import DefaultHeader from '../../components/DefaultHeader'
 
 function LandingPage() {
     return (
         <div>
-            
+            <DefaultHeader/>
```

Comment:
landing page should render the content of the landing page and components connected to it.

all the component `Dialog` `Video` etc. are in `DefaultHeader` which doesn't really makes sense, since that component is supposed to be just the header and not the body of the entire page